testing/test.js

```diff
@@ -0,0 +1,68 @@
+var user = require('../models/User');
+var posts = require('../models/Posts');
+var expect = require('chai').expect;
+var mongoose = require('mongoose');
+
+describe('database `user Model`',function(){
+  it('should be a Mongoose model',function(){
+    expect(new user()).to.be.instanceOf(mongoose.Model);
```

Comment:
Always keep the names of models, in this case "user" capitalized. Should be "User" here.

Reply:
Thank you Jack, I will fix it